prober/tcp.go

```diff
@@ -91,11 +91,11 @@ func dialTCP(ctx context.Context, target string, module config.Module, registry
 func ProbeTCP(ctx context.Context, target string, module config.Module, registry *prometheus.Registry, logger log.Logger) bool {
 	probeSSLEarliestCertExpiry := prometheus.NewGauge(prometheus.GaugeOpts{
 		Name: "probe_ssl_earliest_cert_expiry",
-		Help: "Returns earliest SSL cert expiry date",
+		Help: "Returns earliest SSL cert expiry in unixtime",
```

Comment:
If we want these to be consistent, we should move these help strings to `const` values in `prober.go`.

Or maybe move the whole `prometheus.GaugeOpts{}` to be a package level var.

Reply:
pushed new commit with `const` values in `prober.go`